### DISCUSSION TITLE
How to re-select Spectacle config on Rectangle Pro?

### DISCUSSION THREAD

**Discussion by michael-cohen-io** [rxhanson/RectanglePro-Community#492] [Q&A] [answered]
I accidentally continued past the welcome screen before selecting the Spectacle-style configuration and keyboard shortcuts. I've tried searching around preferences and re-installing but cannot seem to be able to get back to that welcome screen. Any suggestions would be great

**Comment by rxhanson** [comment, marked as answer]
This isn't currently in the UI for Rectangle Pro.

To get Spectacle defaults, you can execute the following terminal command, and restart the app:

```
defaults write com.knollsoft.Hookshot alternateDefaultShortcuts -bool true
```

You can go back to the other set of defaults by setting that to false.

https://github.com/rxhanson/RectanglePro-Community/discussions/181

**Reply by michael-cohen-io** [reply]
Thank you so much! sorry for missing this previous answer!
